CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
  * [`changed`]  Functions are now dedicated per sensor (e.g. `scd30_probe()`
                 instead of `scd_probe()`)
  * [`changed`]  Move the i2c init call out of `probe()` and into the example
+ * [`added`]    Add `scd30_read_serial()` to read out the serial number
 
```

Comment:
Did the embedded-common upgrade also change something worth for the CHANGELOG?

Reply:
multi-bus support?

> git log a1bf1f4691a2e173043de59f720d1f1b999c5288.. --oneline --no-merges 
0f9dd7c (HEAD -> master, github/master, github/HEAD) Use stdlib.h for a C++ compatible NULL definition (#27)
af3f0ce Add STS to list of projects, update doc
e9a61b5 Return 0xFF when i2c_read_byte fails
f05c7ae Fix sign conversion warnings
7611316 Add multi-bus example for Arduino MKRZero
9310b80 Add sensirion_i2c_select_bus for multi-bus use-cases

Reply:
Yes I guess multi-bus is interesting enough for the end users.

Reply:
fixed